for-posoperators/market-de-fair-use-policy.md

```diff
@@ -6,12 +6,16 @@ D-10117 Berlin<br />
 
 
 
-## Verwendung der TSE-as-a-Service
+# Verwendung des fiskaltrust.Sorglos Produkt-Bundles
 
-Kunden, die eine Hardware- oder Cloud TSE im fiskaltrust.Sorglos mit TSE-Bundle als TSE-as-a-Service erhalten haben, können diese gemäß unserer Fair Use Policy nutzen. Dies gilt auch für Kunden, die eine fiskaly Cloud-TSE über den fiskaltrust.Shop erworben haben. Nicht angewendet wird dieses Prinzip für Kunden, die eine fiskaly Cloud-TSE über andere Kanäle als den fiskaltrust.Shop bezogen haben.
+Kunden, die ein fiskaltrust.Sorglos ohne TSE oder fiskaltrust.Sorglos mit TSE Produkt-Bundle erworben haben, können dieses gemäß unserer Fair Use Policy nutzen. 
```

Comment:
Welche fair-use-policy gilt, wenn nur eine cloud-TSE genutzt wird?
Bei Hardware-TSE wird es keine fair-use-policy geben?
In AT gibt es auch Limitierungen für die ft.MW (zB Maximale Anzahl der queues).
> Gibt es hierzu weitere Dokumente?

Reply:
Ich glaube @aedler-ft bezieht sich hier nur auf die TSE-as-a-Service, nicht auf das fiskaltrust.Sorglos an sich?

Reply:
fair use policy gilt derzeit ausschliesslich für die Sorglos Bundles. Einzelprodukte im resale (z.B. fiskaly Cloud TSE) unterliegen den Konditionen des jeweiligen Herstellers. Ich habe die fair use policy entsprechend aktualisiert damit das klarer wird.